seed/urls.py

```diff
@@ -4,3 +4,13 @@
 :copyright (c) 2014 - 2017, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
 :author
 """
+
```

Comment:
file was moved. There used to be a seed/urls/*.py folder. That is gone now. It is just seed/urls.py because, well, the majority of the app is an API.